test/components/board/boardHelper.test.js

```diff
@@ -0,0 +1,7 @@
+import * as boardHelper from '../../../src/components/board/boardHelper';
```

Comment:
If we're only using one of the functions from this file, do we need to import everything? 

We could also 
`import { isMoveAvailable } from '../../../src(etc)'`